src/css/doc.css

```diff
@@ -147,6 +147,16 @@
   margin-top: var(--base-large-space);
 }
 
+/**
```

Comment:
Don't think we need two `*` for multi line comment, just one no?
```suggestion
/*
```

Reply:
Mmm. Seems to be some sort of IntelliJ preference. Yup; it's fine with just the single asterisk.